src/explore-education-statistics-common/src/modules/table-tool/components/FiltersForm.tsx

```diff
@@ -51,31 +51,57 @@ const FiltersForm = (props: Props & InjectedWizardProps) => {
     isActive,
   } = props;
 
+  // Automatically select filter when one filter group with one option
+  const filterEntries = Object.entries(subjectMeta.filters);
+  const autoSelectFilter =
+    filterEntries.length === 1 &&
+    Object.entries(filterEntries[0][1].options).length === 1 &&
```

Comment:
We probably want to avoid excessively using `Object.entries` like this as I'm pretty sure that it's not a super cheap method to use (probably because it has to allocate a lot of arrays for all the entry pairs). 

As mentioned earlier, we should use `Object.values` instead, but would suggest that we allocate some additional variables for these intermediary values too. Looks like we'd be able to use them within `initialFormValues` as well.